pycytominer/cyto_utils/output.py

```diff
@@ -79,6 +79,10 @@ def output(
     )
     """
 
+    # ensure a default output type
+    if output_type is None:
+        output_type = "csv"
```

Comment:
This feels dangerous - perhaps we could throw a warning that None is not supported?

Reply:
ah, I should clarify what I mean by "dangerous" - if a user explicitly sets `output_type=None` and expects certain behavior (e.g., outputting to variable and not writing), they may be surprised that we overwrite it to `output_type="csv"`. Perhaps the solution isn't to throw a warning (maybe add to docstring)? Or, we could elect to do nothing if this is standard expected behavior.

Reply:
Thanks for these comments! I had to make this change in order to integrate `aggregate` with `output` properly given its default parameter types. I recognize we could change the defaults one direction or the other (with `aggregate` or `output`) but I didn't want to move too much outside of the scope of the changes here in order to get a first pass with `mypy`. I wasn't sure about rationalizing a default value for `aggregate`'s `output_type` if it wasn't going to use output. At the same time, I didn't want to change too much of `output`'s default capabilities due to how it integrates with many other functions (could get complex for the scope of this PR).

My feeling here was that if `output` was used at all that we'd expect to be "outputting" something, and that if None were passed to it for `output_type` (especially for nested function calls) that it should revert to the default string value for the parameter in `output`.